web/libs/editor/src/components/Timeline/Views/Frames/Frames.tsx

```diff
@@ -204,22 +212,34 @@ export const Frames: FC<TimelineViewProps> = ({
       };
 
       const offset = getMouseToFrame(e);
+      const baseFrame = toSteps(offset, step) + 1;
 
       setIndicatorOffset(offset);
 
+      if (isDrawing) {
+        region = props.onStartDrawing?.(baseFrame);
+      }
+
       const onMouseMove = (e: globalThis.MouseEvent) => {
         const offset = getMouseToFrame(e);
+        const frame = toSteps(offset, step) + 1;
 
         if (offset >= 0 && offset <= rightLimit) {
           setHoverEnabled(false);
           setRegionSelectionDisabled(true);
           setIndicatorOffset(offset);
         }
+
+        if (region) {
+          const [start, end] = frame > baseFrame ? [baseFrame, frame] : [frame, baseFrame];
+          region.setSequence([{ frame: start, enabled: true }, { frame: end, enabled: false }]);
```

Comment:
It could be not a `TimelineRegion` so `setSequence` could not exist. (At least it is so when we have mixed config, like `VideoRectangle` + `TimelineLabels`)